### ISSUE TITLE
Docker installation not starting

### ISSUE THREAD

**Issue by grillsoftware** [dchesterton/texecom2mqtt-hassio#34]
**Describe the bug**
Pulled the image directly to docker and it not starts. Installed using ansible:
- name: Texecom Container (dchesterson)
  become: true
  docker_container:
    name: texecom
    image: dchesterton/texecom2mqtt:latest
    network_mode: host
    state: started
    restart_policy: unless-stopped
    volumes:
      - /var/opt/texecom:/app

**Application version**
latest

**Home Assistant version**
N/A

**Plugin config**
texecom:
    # Required: Texecom panel IP address
    host: {{texecom_panel_ip}}
    # Optional: UDL password programmed in the panel. Note: this is NOT the code used to arm/disarm the panel (default: 1234)
    udl_password: {{texecom_panel_udl_pass}}
    # Optional: port used to connect to the panel (default: 10001)
    port: 10001

mqtt:
    # Optional: broker URL or IP address (default: localhost)
    host: localhost
    # Optional: broker port (default: 1883 or 8883 for TLS connections)
    port: 1883
    # Optional: topic prefix to use (default: texecom2mqtt)
    prefix: texecom2mqtt
    #username: my_user # Optional: broker user (default: none)
    #password: my_password # Optional: broker password (default: none)
    # Optional: client ID (default: random)
    client_id: texecom2mqtt
    # Optional: keepalive in seconds (default: 10)
    keepalive: 30
    # Optional: retain (default: true)
    retain: true
    # Optional: retain on log messages (default: false)
    retain_log: false
    # Optional: QoS (default: 0)
    qos: 2
    #ca: /cert/ca.pem # Optional: CA for TLS connection (default: none)
    #cert: /cert/cert.pem # Optional: certificate for TLS connection (default: none)
    #key: /cert/key.pem # Optional: private key for TLS connection (default: none)

homeassistant:
    # Optional: enable Home Assistant discovery (default: false)
    discovery: true
    # Optional: Home Assistant MQTT topic prefix (default: homeassistant)
    prefix: homeassistant 

**Debug log**
Error: Cannot find module '/app/app/dist/texecom2mqtt'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
internal/modules/cjs/loader.js:969

**Comment by dchesterton** [comment]
The app lives in /app so you'll need to change your volume to be `/var/opt/texecom/config.yml:/app/config.yml` as per the documentation: https://github.com/dchesterton/texecom2mqtt-hassio#docker as you are currently overwriting the whole app volume.

**Comment by grillsoftware** [comment]
thanks, changing the volume did work. I'm new to all this and didn't realized that would overwrite the whole volume.

That was fixed but I still have an issue

**Debug Log**
(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1),
(node:1) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.,
(node:1) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'match' of undefined,
    at /snapshot/app/node_modules/better-ajv-errors/lib/modern/helpers.js:26:53,
    at Array.forEach (<anonymous>),
    at makeTree (/snapshot/app/node_modules/better-ajv-errors/lib/modern/helpers.js:21:13),
    at _default (/snapshot/app/node_modules/better-ajv-errors/lib/modern/helpers.js:116:16),
    at _default (/snapshot/app/node_modules/better-ajv-errors/lib/modern/index.js:27:45),
    at validate (/snapshot/app/dist/config.js:58:24),
    at Object.loadConfig (/snapshot/app/dist/config.js:41:16),
    at /snapshot/app/dist/index.js:11:29,
    at Object.<anonymous> (/snapshot/app/dist/index.js:74:3),
    at Module._compile (pkg/prelude/bootstrap.js:1433:22)

I looked in the closed issues and found one with the same error message. It was fixed by mapping the correct arming modes of the areas. However, I have no areas or zones configured yet as the documentation states they are optional.
https://github.com/dchesterton/texecom2mqtt-hassio/issues/30

I tried adding areas: [] and zones: [] but it didn't help.

I have no visual on the Texecom panel, but if there are details needed to troubleshoot I can ask to my on-site team to send whatever is needed.
